orc8r/cloud/go/services/orchestrator/servicers/gaugeconverter.go

```diff
@@ -111,10 +109,16 @@ func histogramToGauges(family *prometheus_models.MetricFamily) []*prometheus_mod
 		}
 		countFamily.Metric = append(countFamily.Metric, &countMetric)
 
+		baseLabels := make([]*prometheus_models.LabelPair, len(metric.GetLabel()))
+		for idx, l := range metric.GetLabel() {
+			copyL := *l
+			baseLabels[idx] = &copyL
+		}
+
```

Comment:
I didn't have problems with Histograms but I assumed the same thing could happen in the future so I decided to make it consistent in both since they're essentially the same (labels are just named differently)